git_flow.rst

```diff
@@ -0,0 +1,189 @@
+.. -*- mode: rst -*-
+
+.. role:: bash(code)
+   :language: bash
+
+Recommended GitHub workflow
```

Comment:
I think it should be a separate page. I'd prefer multiple small pages to one big page. People don't like reading big pages generally. Plus this is a very general problem and I intend to point people to this page from multiple projects.